app/src/main/java/com/sun_asterisk/moviedb_50/data/source/local/LocalDataSource.kt

```diff
@@ -0,0 +1,13 @@
+package com.sun_asterisk.moviedb_50.data.source.local
+
+import com.sun_asterisk.moviedb_50.data.source.DataSource
+
+object LocalDataSource : DataSource.LocalDataSource {
```

Comment:
Tất cả dòng dưới vô nghĩa, chỉ cần dòng đầu thôi

Reply:
https://kotlinlang.org/docs/tutorials/kotlin-for-py/objects-and-companion-objects.html#object-declarations
If you need a singleton - a class that only has got one instance - you can declare the class in the usual way, but use the object keyword instead of class:
``` Kotlin
object CarFactory {
    val cars = mutableListOf<Car>()
    
    fun makeCar(horsepowers: Int): Car {
        val car = Car(horsepowers)
        cars.add(car)
        return car
    }
}
```